receiver/hostmetricsreceiver/internal/scraper/memoryscraper/memory_scraper_linux.go

```diff
@@ -7,13 +7,29 @@ package memoryscraper // import "github.com/open-telemetry/opentelemetry-collect
 
 import (
 	"github.com/shirou/gopsutil/v4/mem"
+	"go.opentelemetry.io/collector/featuregate"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/memoryscraper/internal/metadata"
 )
 
+var useMemAvailable = featuregate.GlobalRegistry().MustRegister(
+	"receiver.hostmetricsreceiver.UseLinuxMemAvailable",
+	featuregate.StageAlpha,
```

Comment:
Looks like we already moved to the [updated](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/42409) Used from gopsutil. So we don't need this PR anymore. Or we can make it beta to not get back